strongbox-commons/src/main/java/org/carlspring/strongbox/booters/SystemPropertiesInitializer.java

```diff
@@ -0,0 +1,56 @@
+package org.carlspring.strongbox.booters;
+
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+/**
+ * Ensures that a handful of core system properties are set prior to starting the application.
+ *
+ * @author cbono
+ */
+class SystemPropertiesInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext>
```

Comment:
@bono007 seems we don't need any of this properties to be initialized as System properties, so we don't need this `SystemPropertiesInitializer`.